docs/api/doc-block-controls.md

```diff
@@ -31,7 +31,7 @@ import * as ButtonStories from './Button.stories'
 
 <Callout variant="warning">
 
-The Controls doc block will only have functioning UI controls if you have also installed and registered [`@storybook/addon-controls`](../essentials/controls.md) (included in [`@storybook/addon-essentials`](../essentials/index.md)).
+The Controls doc block will only have functioning UI controls if you have also installed and registered [`@storybook/addon-controls`](../essentials/controls.md) (included in [`@storybook/addon-essentials`](../essentials/index.md)) and haven't turned off inline stories with the [`inlineStories`](./doc-block-story.md#inline) configuration option.
```

Comment:
It's confusing to reference a non-existent property with the code formatting. The preceding "inline stories" adds enough clarity, I think.

```suggestion
The Controls doc block will only have functioning UI controls if you have also installed and registered [`@storybook/addon-controls`](../essentials/controls.md) (included in [`@storybook/addon-essentials`](../essentials/index.md)) and haven't turned off inline stories with the [`inline`](./doc-block-story.md#inline) configuration option.
```